api/WindowEventHandlers.json

```diff
@@ -642,39 +659,56 @@
               "version_added": "49"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "67"
             },
             "edge": {
-              "version_added": true
+              "version_added": null
             },
             "edge_mobile": {
-              "version_added": true
+              "version_added": null
             },
             "firefox": {
               "version_added": false,
```

Comment:
I guess we should set this to version 55 and `"partial_implementation": true`: https://bugzilla.mozilla.org/show_bug.cgi?id=1338059